### PR TITLE
Add wheel builder information

### LINKED ISSUE #4: Grab more wheel metadata
Check .dist-info and see what can be grabbed.

### PR DESCRIPTION
**Pull request by ofek**
Resolves #4 using https://github.com/pypi/inspector

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -211,8 +219,8 @@ def get_metadata_by_install(package, resp):
 
     print(f"building {package!r} from source!")
     popen = subprocess.Popen(
-        f"{venv_python} -c 'import json; from importlib_metadata import requires, metadata; "
-        f'package="{package}"; print(json.dumps({{"requires_dist": requires(package), "requires_python": metadata(package).get("Requires-Python", "")}}))\'',
+        f"{venv_python} -c 'import json; from importlib_metadata import Distribution; "
+        f'd=Distribution.from_name("{package}"); print(json.dumps({{"requires_dist": d.requires(package), "requires_python": d.metadata.get("Requires-Python", ""), "wheel_data": dist.read_text("WHEEL") or ""}}))\'',
```
Comment (ofek) [inline main.py:223]: This function isn't used but I thought I'd be a good citizen 🙂

**Review by sethmlarson — APPROVED** [review]
LGTM, thank you!!

**Comment by ofek** [comment]
Did the job finish yet?

**Comment by sethmlarson** [comment]
@ofek Unfortunately the current implementation hammering that service caused it to have a lot of stability issues (5XXs, timeouts, etc) so I decided we shouldn't be using that method. Might have to download and inspect one wheel per package?

**Comment by ofek** [comment]
Sorry, opened https://github.com/sethmlarson/pypi-data/pull/9

**Pull request merged**